ci/prepare-documentation.py

```diff
@@ -42,7 +42,7 @@
 import sys
 import urllib.parse
 import urllib.request
-import xml.etree.ElementTree as ET
+import xml.etree.ElementTree as ET # nosec - disable B405:import-xml-etree check
```

Comment:
The recommendation here is to replace `xml.etree.ElementTree` with the equivalent `defusedxml` package. However, the  `defusedxml` doesn't contain `Element` and `SubElement` classes or equivalent one, therefore this replacement would require changes in `add_page` implementation. 
For now the implementation with `xml.etree` is left.

Reply:
Ok